zulipterminal/ui_tools/views.py

```diff
@@ -558,6 +558,10 @@ def streams_view(self) -> Any:
                     count=self.model.unread_counts['streams'].get(stream[1], 0)
                 ) for stream in self.view.unpinned_streams]
 
+        self.model.stream_id_to_button = {stream.stream_id: stream for
+                                          stream in streams_btn_list if
+                                          hasattr(stream, 'stream_id')}
+
```

Comment:
Yeah it does. but it's okay I suppose for now.

Reply:
Agreed, I have added it to the UI for now. Controller is another good place, but as Neil pointed out since I store this during a UI building phase, it would good to keep store in the UI itself.

Reply:
Sure.+1: